README.md

```diff
@@ -66,27 +63,18 @@ Please follow [Getting Started][gs] to get your hands on Open-SIR.
 
 ## Authors
 
-* **[José Álamos][alamos]** -
-  [RIOT-OS][riot]
-* **[Felipe Huerta][huerta]** - [PhD
-  Student][phd-huerta] at [Imperial
-College London][imperial]
-* **[Sebastián Salata][salata]** - Software Engineer -
```

Comment:
Sorry. I really appreciate being named as an author here, but I am not. I am a contributor (I haven't touched almost anything in the python codebase).

Reply:
As long as you feel comfortable, I'm OK with that :raised_hands: